style.css

```diff
@@ -29,6 +32,117 @@ img {
     margin-bottom: 0.5em;
 }
 
+body svg {
+    box-sizing: initial;
+}
+
+/* navbar */
+
+nav {
+    background-color: var(--white)
+}
+
+.text-input {
+    background-color: var(--gray-1);
+    border-radius: 2em;
+    padding-top: 0.8em;
+    padding-bottom: 0.8em;
+    font-size: 0.9em;
+    border: 0;
+    padding-left: 2.5em;
+}
+
+
+svg {
```

Comment:
This looks weird... Are we sure we want ALL `svg` elements in the entire website to be gray, and `1em` width?